pkg/agent/plugin/workloadattestor/k8s/k8s_posix.go

```diff
@@ -199,8 +224,17 @@ func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv1.AttestReque
 			status, lookup := lookUpContainerInPod(containerID, item.Status)
 			switch lookup {
 			case containerInPod:
+				selectors := getSelectorValuesFromPodInfo(&item, status)
+				log.Debug("Attemping to get signature info from image", status.Name)
+				sigstoreSelectors, err := p.sigstore.AttestContainerSignatures(ctx, status)
```

Comment:
@willallves something that we discussed in the last maintainer's sync is that it would be prudent to have this feature enabled under an experimental configuration, making this an [experimental feature](https://github.com/spiffe/spire/blob/main/doc/upgrading.md#experimental-features).
The main motivation for this is that we don't really know how the changes to support the validation of images signatures through sigstore would work with existing deployments in the real world.
Could you please have the `sigstore {...}` config under an `experimental {...}` section?